assets/styles/app.scss

```diff
@@ -19,3 +26,34 @@ body {
     text-align: center;
     padding: 2rem;
 }
+
+// ---------------------------------Landing page------------------------------------ //
+
+.landing-page{
+    display: flex;
+    flex-direction: column;
+}
+
+.landing-page .logo-container{
+    margin-bottom: 4rem;
+    text-align: center;
+}
+
+.landing-page .logo{
+    width: 100%;
+}
+
+.landing-button-container{
+    text-align: center;
+}
+
+.landing-button{
+    background-color: $color-3;
+    font-size: 1em;
+    border-radius: 2rem;
+    width: 60vw;
+    height: 10vh;
```

Comment:
buttons should not have a fixed size: use layout or spacing if you need it to take more space